packages/cra-template-mephisto-review/template/src/components/AllItemView.jsx

```diff
@@ -10,10 +10,10 @@ import {
   NavbarHeading,
   Alignment,
 } from "@blueprintjs/core";
-import { Tooltip2 } from "@blueprintjs/popover2";
-import { DefaultItemRenderer } from "./plugins/DefaultItemRenderer";
-import { DefaultItemListRenderer } from "./plugins/DefaultItemListRenderer";
-import { Pagination } from "./components/Pagination";
+import { Tooltip } from "@blueprintjs/core";
```

Comment:
I think we can just use Tooltip here instead of Tooltip2 and avoid the additional dependency. In the future we'll migrate to @blueprintjs/core v4 anyway.